deployments/helm-chart/templates/controller-daemonset.yaml

```
@@ -67,6 +67,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        resources:
+{{ toYaml .Values.controller.resources | indent 12 }}
```

After https://github.com/nginxinc/kubernetes-ingress/pull/636#discussion_r307244549 this indentation needs to be adjusted
```suggestion
{{ toYaml .Values.controller.resources | indent 10 }}
```

---

Done!